File: 2-semestre/uri-online-judge/c/1117.c
#include <stdio.h>
 
int main() {
    int contagem = 0;
    double nota, media = 0;

    while (contagem != 2) {
        scanf("%lf", &nota);

        if (nota >= 0 && nota <= 10) {
            media += nota;
            contagem++;
        } else printf("nota invalida\n");

        if (contagem == 2) printf("media = %.2f\n", media/2);
    }
 
    return 0;
}

File: 2-semestre/uri-online-judge/javascript/1117.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let contagem = 0, media = 0;

while (contagem != 2) {
    let nota = parseFloat(lines.shift());

    if (nota >= 0 && nota <= 10) {
        media += nota;
        contagem++;
    } else console.log("nota invalida");

    if (contagem === 2) console.log(`media = ${media/2}`);
}
